### ISSUE TITLE
validate Heroku to enable the mandrill and memcachier addons

### ISSUE THREAD

**Issue by eddroid** [Code-for-Miami/ohana-api#15]
This app tries to use Mandrill (for email) and Memcachier (for caching), but neither can be enabled unless the Heroku account is validated with a credit card.

Lacking Memcachier is not big deal. The app is slower and the logs are noisy.

Lacking Mandrill means we can't send emails to manage user registration. This makes it harder for people to contribute service data.

I don't know what the CfM policy is for Heroku apps that require credit card validation.

**Comment by greggish** [comment]
I do think that we should be hosting this on CfM's Heroku account. That said, I wouldn't assume at this point that we're going to really try to get organizations to manage their information in Ohana... I think many organizations do manage their information through Switchboard directly, and the main objective of this Ohana API and Web Search is to demonstrate the basic concepts of how an API works and why it makes it easier to develop quick and context-specific websites that use this important data. 

In other words, lacking Mandrill might not be a big deal...
